Makefile

```diff
@@ -158,19 +165,18 @@ format: ; $(info $(M) running format )
 
 .PHONY: help
 help: ## Shows this help message
-	$(Q) echo 'usage: make [target] ...'
-	$(Q) echo
-	$(Q) echo 'targets : '
-	$(Q) echo
-	$(Q) fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'| column -s: -t
+	$(Q) awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n\nTargets:\n"} /^[a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m\t %s\n", $$1, $$2 }' $(MAKEFILE_LIST)
 
-### Dependencies
+# Dependencies
 
 $(GOTEST_BIN): ; $(info $(M) getting gotest )
-	$(Q) GO111MODULE=off $(GOGET) -u github.com/rakyll/gotest
+	$(Q) $(GOBUILD) -o $@ github.com/rakyll/gotest
 
 $(EMBEDMD_BIN): ; $(info $(M) getting embedmd )
-	$(Q) GO111MODULE=off $(GOGET) -u github.com/campoy/embedmd
+	$(Q) $(GOBUILD) -o $@ github.com/campoy/embedmd
+
+$(LICHE_BIN): ; $(info $(M) getting liche )
+	$(Q) $(GOBUILD) -o $@ github.com/raviqqe/liche
```

Comment:
Another alternative for installing tools : 
```
# List of tools
GOTOOLS := $(shell cat tools.go | grep _ | awk -F'"' '{print $$2}')

.PHONY: install-tools
install-tools: ## Install tools
install-tools: vendor ; $(info $(M) installing tools)
	$(Q) $(GOINSTALL) $(GOTOOLS)
```

Reply:
👍